apb.yml

```diff
@@ -33,12 +33,22 @@ plans:
       title: App target
       type: enum
       default: "Android"
-      enum: ["Android"]
+      enum: ["Android", "IOS"]
     - name: googlekey
-      required: True
       title: Your Google Key for Firebase Cloud Messaging
       type: string
+      display_group: Android
     - name: projectNumber
-      required: True
       title: Your FCM Project Number, needed on the mobile client for connecting with FCM
       type: string
+      display_group: Android
+    - name: cert
+      title: iOS .p12 file
```

Comment:
should we say that this needs to be base64 encoded?

Reply:
make a suggestion